hackathon_site/registration/jinja2/registration/test_emails/test_activation_email_subject.txt

```diff
@@ -0,0 +1,3 @@
+Hello there,
+
+This template is used as part of the unit tests. It can be ignored.
```

Comment:
Should the subject be 3 lines?

Reply:
Not that it matters tho since you join it into a single line later

Reply:
That's exactly why this one is 3 lines, so that the test can make sure it becomes 1